File: solutions/java/resistor-color-duo/2/src/main/java/ResistorColorDuo.java
import java.util.List;

class ResistorColorDuo {
    private final List<String> r = List.of(
        "black","brown","red","orange","yellow",
        "green","blue","violet","grey","white"
    );

    int value(String[] c) {
        return r.indexOf(c[0]) * 10 + r.indexOf(c[1]);
    }
}

File: solutions/ruby/resistor-color-duo/2/resistor_color_duo.rb
=begin
Write your code for the 'Resistor Color Duo' exercise in this file. Make the tests in
`resistor_color_duo_test.rb` pass.

To get started with TDD, see the `README.md` file in your
`ruby/resistor-color-duo` directory.
=end
module ResistorColorDuo
  COLORS = %w[black brown red orange yellow green blue violet grey white]
  def self.value(colors)
    COLORS.index(colors[0]) * 10 + COLORS.index(colors[1])
  end
end
